### PR TITLE
Refactoring code into multiple files

### LINKED ISSUE #14: Refactor project files
Resort project files into their appropriate locations, make sure to update the `npm start` command in package.json when doing completing this task so we can still use this command.

### PR DESCRIPTION
**Pull request by Vic-ST**
Refactored all commands into multiple files in the commands directory, looks something like this in VS Code:

image

Works well, tested each command.

Closes #14

### CODE REVIEW

**Review by fedGL — APPROVED** [review]
LGTM. Follows the pattern explained on the tutorial, all files seem well-formed

File: commands/discourse/verify.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+module.exports = {
+    name: 'verify',
+    description: 'Verifies any the user ID you provided',
```
Comment (aedwardg) [inline commands/discourse/verify.js:3]: Let's get rid of "any" here

File: app.js [hunk 2]
```diff
@@ -24,294 +37,37 @@ client.on('ready', () => {
...
-  *Admin Only`);
-}
-
-
-
 const commandParser = (msg) => {
   splitMessage = msg.content.split(" ")[0];
+  
+	const args = msg.content.slice("!".length).trim().split(/ +/);
+	const command = args.shift().toLowerCase();
```
Comment (aedwardg) [inline app.js:44]: `splitMessage` isn't needed (or used) anymore because of the new way to parse the args.  Let's get rid of it.

**Review by aedwardg — COMMENTED** [review]
@Vic-ST just 2 quick changes and then I think this is ready to merge!

There's still some formatting inconsistencies here and there, but I think it's best to set up the pre-commit hook and let that fix them for us instead of trying to find them by hand.

I also still think that we could move a couple of the imports into the command files:

- colors (from constants.js) could be moved to only be imported in `createroles.js`
- uuidv4 and sendgrid could be only imported in `sendcode.js` 

Then they wouldn't have to be passed in as parameters to those command functions. That should probably be in a separate issue/PR though.

**Comment by Vic-ST** [comment]
@aedwardg Should all be fixed!

**Review by aedwardg — APPROVED** [review]
LGTM :rocket:

**Pull request merged**